code/game/objects/items/storage/belt.dm

```diff
@@ -16,6 +16,20 @@
 	///TRUE Means that it closes a flap over its contents, and therefore update_icon should lift that flap when opened. If it doesn't have _half and _full iconstates, this doesn't matter either way.
 	var/flap = TRUE
 
+/obj/item/storage/belt/gun/flaregun/dump_into(obj/item/storage/Main, mob/user)
+
+	if(length(holstered_guns) < 1 && length(contents) >= (storage_slots-1))
+
+		to_chat(user, SPAN_WARNING("[src] is full."))
+		return FALSE
+	return ..()
+
+/obj/item/storage/belt/gun/flaregun/handle_item_insertion(obj/item/Warn, prevent_warning = 0, mob/user)
```

Comment:
Three things:
- `Warn` makes no sense to be the variable name; this is kinda more confusing than just a random letter like it used to be.
- We use snake_case; not camelCase or PascalCase. Your variable here is PascalCase. https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#type-paths-must-be-snake-case
- Use `FALSE` instead of `0` for booleans.

So something more like `obj/item/new_item` would be more appropriate.

Reply:
i replace the Warn by new_item and replace the case of = 0 with = FALSE